ops/rackspace-agent-plugins/bf-rollups-delay.py

```diff
@@ -1,4 +1,6 @@
-#! /usr/bin/python
+#!/usr/bin/env python
+'''Blueflood Rollup Delay'''
+'''For each rollup level, lists the number of slots which need to processed by blueflood.  One day is approximately 300 slots.'''
```

Comment:
A queue of 300 slots to-be-processed is equivalent of 1 day's worth of processing.  I.e., the system is a day behind.  Is that right @GeorgeJahad ?